src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -239,9 +257,14 @@ private function getLocalFile(string $path)
     private function getRemoteFile(string $remoteRepository, string $path)
     {
         $path = \str_replace(\DIRECTORY_SEPARATOR, '/', $path);
-        $response = $this->httpClient->request('GET', $remoteRepository . $path);
 
-        return $response->getContent();
+        try {
+            $response = $this->httpClient->request('GET', $remoteRepository . $path);
+
+            return $response->getContent();
+        } catch (ClientException $e) {
```

Comment:
can we catch a more specific exception here? 🤔 
think this could lead to confusing behaviour if an error happens because of an unreliable internet connection

Reply:
The represents 4xx status code.